Automattic/jetpack
# Social: Implement StatCards component

This PR replaces the current inline "stat card" on the Jetpack Social admin page (the card that shows the number of shares) with a more modular component, that can be used for other cards as well. It also slightly improves the styling, and prevents the card from changing size after loading.

#### Changes proposed in this Pull Request:

* Replace the existing stat card with a modular component.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure you have a social connection.
* Go to the Jetpack Social admin page.
* Check that the card showing the amount of shares displays properly.


https://user-images.githubusercontent.com/1713699/179474479-c8a9b9eb-eaf3-459d-b737-2d04c04fd585.mp4
